_posts/2022-07-19-snapshots.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title:  "Snapshots"
+lang: en
+categories: Documentation
+excerpt: "Snapshots allows users to preserve vocabulary state in time."
+---
+
+[🇨🇿 Czech version]({{"/cs/dokumentace/snapshots" | relative_url}}){: .btn .btn--info}
+
+Snapshots allows users to preserve vocabulary state in time.
+
+Snapshot can be created by clicking Create snapshot button in vocabulary details.
+
+{% include figure image_path="/assets/images/posts/documentation/snapshot-1.png" alt="Snapshot can be created by clicking Create snapshot button in vocabulary details." %}{: .post-image }
+
+Vocabulary snapshot is ontologically both vocabulary and snapshot. It contains all tropes and relators as vocabulary plus relation to the original vocabulary and date and time in which was snapshot created.
+
+Together with vocabulary snapshot are created snapshots of all terms from the vocabulary. Same as for vocabulary snapshot, term snapshot is both snapshot and term with relation to original term and vocabulary snapshot and containing date and time of snapshot creation (which is the same as vocabulary snapshot creation date and time).
+
+{% include figure image_path="/assets/images/posts/documentation/snapshot-2.png" alt="Schema of snapshots." %}{: .post-image }
+
+Create snapshot button creates also snapshots of all terms related to the terms from the original vocabulary and their vocabularies. Instead of links between terms, snapshot therefore contains links between term snapshots (which have link to their vocabulary snapshot) from the exact time when the snapshot was created.
```

Comment:
Snapshot creation is automatically cascaded to all (transitively) related vocabularies (vocabularies that contain terms in SKOS relationships with terms from the initial vocabulary).